lib/actions/push.ts

```diff
@@ -98,42 +96,6 @@ async function getAppOwner(sdk: BalenaSDK, appName: string) {
 	return selected.extra;
 }
 
-interface RegistrySecrets {
-	[registryAddress: string]: {
-		username: string;
-		password: string;
-	};
-}
-
-async function parseRegistrySecrets(
-	secretsFilename: string,
-): Promise<RegistrySecrets> {
```

Comment:
This function was moved from `push.ts` to `compose_ts.ts` because it is now shared and used by the three commands: push, build and deploy.